.env

```
@@ -1,10 +1,10 @@
 # Common Environment Variables
-NEXT_PUBLIC_YORKIE_VERSION='0.7.0'
-NEXT_PUBLIC_YORKIE_JS_VERSION='0.7.0'
+NEXT_PUBLIC_YORKIE_VERSION='0.7.1'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)